EasyPost.EasyVCR/Mode.cs

```diff
@@ -0,0 +1,25 @@
+namespace EasyPost.EasyVCR
+{
+    /// <summary>
+    ///     Modes of operation for the VCR system.
+    /// </summary>
+    public enum Mode
+    {
+        /// <summary>
+        ///     No recording or playback. Make HTTP requests as normal.
+        /// </summary>
+        Bypass,
+        /// <summary>
+        ///     Replay existing recordings if they exist, otherwise record new ones.
+        /// </summary>
+        Auto,
```

Comment:
I can't tell yet, is `Auto` the default behavior? If not, it should be to match other VCR solutions.

Since this is a list of sorts, could we sort it alphabetically?

Reply:
Yes, `Auto` is the default for the VCR. For a standalone HttpClient, there is no default mode; you have to explicitly say what mode you want.

I'll sort the list.